server/app/services/export/JsonExporter.java

```diff
@@ -168,92 +169,47 @@ private CfJsonDocumentContext buildSingleApplicationJson(
             .addApplicationEntries(entriesBuilder.build())
             .build();
 
-    return buildSingleApplicationJson(applicationJsonExportData);
```

Comment:
Instead of nesting this call inside of a different `buildSingleApplicationJson()` (this takes an `ApplicationExportData` and the outer one takes an `Application`), we rename this to `buildApplicationExportData()` and call it from the same callsite as the outer one (`exportPage()`)